src/stores/tag-store.ts

```diff
@@ -41,9 +41,10 @@ import { TagNode } from '@/types';
 interface StoreState {
   data: Record<string, TagNode>;
   fetchData: () => Promise<void>;
+  setData: (tagCounts: object) => void;
 }
 
```

Comment:
mostly this splits fetchData into two functions, so that the second can be called by blueprint store